### PR TITLE
fix basic widgets tutorial example

### PR DESCRIPTION
**Pull request by sethladd**
Addresses https://github.com/flutter/flutter.github.io/issues/34

### CODE REVIEW

**Comment by sethladd** [comment]
thanks for taking a look!

File: tutorial.md [hunk 1]
```diff
@@ -104,7 +104,16 @@ class MyScaffold extends StatelessComponent {
   }
 }
 
-void main() => runApp(new MyScaffold());
+void main() {
+  runApp(
+    new MaterialApp(
+      title: 'My app',
+      routes: {
```
Comment (Hixie) [inline tutorial.md]: `routes: <String, RouteBuilder>{`

File: tutorial.md [hunk 2]
```diff
@@ -104,7 +104,16 @@ class MyScaffold extends StatelessComponent {
   }
 }
 
-void main() => runApp(new MyScaffold());
+void main() {
+  runApp(
+    new MaterialApp(
```
Comment (Hixie) [inline tutorial.md]: you can put the 'new MaterialApp(' on the same line as the runApp().

File: tutorial.md [hunk 3]
```diff
@@ -104,7 +104,16 @@ class MyScaffold extends StatelessComponent {
...
+      title: 'My app',
+      routes: {
+        '/': (RouteArguments args) => new MyScaffold()
+      }
+    )
+  );
+}
 ```
 
 The `MyToolBar` component creates a `Container` with a height of 56
```
Comment (Hixie) [inline tutorial.md]: add a paragraph at the end of this subsection that explains the runApp() call

**Comment by Hixie** [comment]
LGTM, modulo comments

File: tutorial.md (hunk 3 above)
Comment (sethladd) [inline tutorial.md]: I'll try to add something, but we'll need to clean this section up. This is from a section called "basic widgets", but now we've introduced MaterialApp and routes.

**Comment by sethladd** [comment]
@Hixie ptal Thanks!

File: tutorial.md [hunk 4]
```diff
@@ -104,9 +104,21 @@ class MyScaffold extends StatelessComponent {
...
+      routes: <String, RouteBuilder>{
+        '/': (RouteArguments args) => new MyScaffold()
+      }
+    )
+  );
+}
 ```
 
+Many widgets need to be inside of a `MaterialApp` to display properly,
+in order to inherit theme data. Therefor, we run the application
```
Comment (abarth) [inline tutorial.md]: Therefor -> Therefore

Comment (sethladd) [inline tutorial.md]: good catch, thanks!

File: tutorial.md [hunk 5]
```diff
@@ -104,9 +104,21 @@ class MyScaffold extends StatelessComponent {
...
+void main() {
+  runApp(new MaterialApp(
+      title: 'My app',
+      routes: <String, RouteBuilder>{
+        '/': (RouteArguments args) => new MyScaffold()
+      }
+    )
+  );
+}
 ```
```
Comment (Hixie) [inline tutorial.md]: Unindent lines 109-112 and put make line 113/114 be just `));`.

**Comment by Hixie** [comment]
LGTM.

**Pull request merged**